packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -364,6 +364,31 @@ Sample user data:
 | Workday ID                                                         | `hasWorkdayUser() ? findWorkdayUser().employeeID : null`                                                                                                 | 123456                  | Check if user has a Workday assignment, and if so, return their Workday employee ID.                                                                                                                                                                                                                                                                                                                   |
 | Active Directory UPN                                               | `hasDirectoryUser() ? findDirectoryUser().managerUpn : null`                                                                                             | bob@okta.com            | Check if user has an Active Directory assignment, and if so, return their Active Directory manager UPN.                                                                                                                                                                                                                                                                                                |
 
+
+## Expressions for custom claims
+
+Okta provides a few expressions that you can only use with custom claims.
+* See [Create claims](/docs/guides/customize-authz-server/create-claims) for general information about claim creation.
+* See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute) for details about using app-specific values.
+
+> **Note:** The application reference is usually the `name` of the application, as distinct from the `label` (display name). See [Application properties](/docs/reference/api/apps/#application-properties). If your organization configures multiple instances of the same application, the names of the later instances are differentiated by a randomly assigned suffix, for example: `zendesk_9ao1g13`.  You can find the name of any specific app instance in the Profile Editor, where it appears in lighter text beneath the label of the app.
```

Comment:
This note suggests you can use explicit references to app, like zendesk_9ao1g13.clientId.

However, earlier in the document there is a note which says `Note: Explicit references to apps aren't supported for custom OAuth/OIDC claims.`

Reply:
At your suggestion, I'll remove the note. Thanks!